dspace-api/src/main/java/org/dspace/app/util/GoogleBitstreamComparator.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.util;
+
+import org.apache.log4j.Logger;
+import org.dspace.content.Bitstream;
+
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by frederic on 24/04/17.
```

Comment:
A minor thing, but please add comments to the header here around the purpose of this new `GoogleBitstreamComparator` class